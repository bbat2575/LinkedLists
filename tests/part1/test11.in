REMOVE 0
NEW 4
1
2.2
E
Four
NEW 1
2e4
REMOVE +1
REMOVE -1
REMOVE +0
REMOVE -0
REMOVE +
REMOVE -
REMOVE 2
REMOVE o
REMOVE O
REMOVE ZERO
REMOVE .
REMOVE 0.
REMOVE .0
REMOVE 0.0
REMOVE 0e0
REMOVE 2e-6
REMOVE 0 0
REMOVE  0
REMOVE   0
REMOVE
REMOVE  
  REMOVE
0REMOVE
0 REMOVE
REMOVE0
 REMOVE 0
  REMOVE 0
REMOV E 0
REM OVE 0
R EMOVE 0
remove 0
Remove 0
ReMoVe 0
REMoVE 0
ERMOVE 0
