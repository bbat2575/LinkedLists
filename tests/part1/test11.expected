
Type HELP for a list of options
Press <CTRL + D> to Quit

> INVALID COMMAND: REMOVE

> List 0: 1 -> 2.20 -> E -> Four

> List 1: 20000.00

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> 
